### ISSUE TITLE
taxonomicName: subgenus without bracket

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#559]
in this case a taxonomic name includes a subgenus, but it is not in brackets

![image](https://user-images.githubusercontent.com/4609956/51031427-d7233100-159c-11e9-8cff-58c220906dab.png)

http://tb.plazi.org/GgServer/summary/0767FFD8082E3F06FFB13B380732F137

**Comment by gsautter** [comment]
Is this even legal under the code? At least it must be so rare we've never had it before ... I recently added the labeled notation (i.e., "Aus subg. Bus cus") for botany, but this ...

I'm afraid relaxing FAT's patterns to cover this case might well incur all sorts of false positives (and other ambiguity issues) in many other places ... just think of treatments switching epithets between genus and subgenus.
To make matters worse, "Caenotingis" occurs with a "subgenus" label only in a single discussion paragraph, which might not even be enough for a reliable inference.

I'll try and think of something, but I'm starting to think this is just one of _those_ articles ... sloppy layout and typos in many key places, etc. ... we have some half dozen tickets about this very article now, do we?

**Comment by myrmoteras** [comment]
I would agree to not cover this case. So when we see it we are aware that this needs some manual work.

**Comment by myrmoteras** [comment]
yes, there are few. This is a combination of being picky in regards of QC and editing

**Comment by gsautter** [comment]
Why do authors vary their notations? I just discovered "Stephanitis (Norba) suthepensis sp. nov." in the usual parenthesis notation, and, expectably, it discovers and parses just fine ...

All the gizmos that I build rely on very little in terms of global rules, large training sets, etc. But the one thing they do rely upon is the existence of consistent patterns _within_ a document, and that's been serving us extremely well for pretty long now.
I used to think copyediting takes care of such variations and outliers ...
